docs/getting-started.md

```diff
@@ -1,168 +1,49 @@
 ---
 id: environment-setup
-title: Setting up the development environment
+title: Get Started with React Native
 hide_table_of_contents: true
 ---
 
-import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import constants from '@site/core/TabsConstants';
+import PlatformSupport from '@site/src/theme/PlatformSupport';
+import BoxLink from '@site/src/theme/BoxLink';
 
-import GuideLinuxAndroid from './\_getting-started-linux-android.md'; import GuideMacOSAndroid from './\_getting-started-macos-android.md'; import GuideWindowsAndroid from './\_getting-started-windows-android.md'; import GuideMacOSIOS from './\_getting-started-macos-ios.md';
+**React Native allows developers who know React to create native apps.** At the same time, native developers can use React Native to gain parity between native platforms by writing common features once.
 
-This page will help you install and build your first React Native app.
+We believe that the best way to experience React Native is through a **Framework**, a toolbox with all the necessaries APIs to let you build production ready apps.
 
-**If you are new to mobile development**, the easiest way to get started is with Expo Go. Expo is a set of tools and services built around React Native and, while it has many [features](https://docs.expo.dev), the most relevant feature for us right now is that it can get you writing a React Native app within minutes. You will only need a recent version of Node.js and a phone or emulator. If you'd like to try out React Native directly in your web browser before installing any tools, you can try out [Snack](https://snack.expo.dev/).
+You can also use React Native without a Framework, however we’ve found that most developers benefit from using a React Native Framework like [Expo](https://expo.dev). Expo provides features like file-based routing, high-quality universal libraries, and the ability to write plugins that modify native code without having to manage native files.
 
-**If you are already familiar with mobile development**, you may want to use React Native CLI. It requires Xcode or Android Studio to get started. If you already have one of these tools installed, you should be able to get up and running within a few minutes. If they are not installed, you should expect to spend about an hour installing and configuring them.
+<details>
+<summary>Can I use React Native without a Framework?</summary>
```

Comment:
```
You can also use React Native without a Framework ...
```

shortly after...

```
<summary>Can I use React Native without a Framework?</summary>
```

it seems like the paragraph beginning with "You can also.." says a lot of the same things as the details/summary section

Reply:
Perhaps we could remove "You can also use React Native without a Framework, however " in the next paragraph, and so it would read like: 

```
We believe that the best way to experience React Native is through a **Framework**, a toolbox with all the necessaries APIs to let you build production ready apps.

We’ve found that most developers benefit from using a React Native Framework like [Expo](https://expo.dev). Expo provides features like file-based routing, high-quality universal libraries, and the ability to write plugins that modify native code without having to manage native files.
```

Reply:
Starting with "Yes" might lend a little clarity, for instance, "Yes, but we believe the best way..."